### PR TITLE
Add new static NPC in city.

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
```SUMMARY: Content "Adds a static npc that spawns in a restaurant.  Adjusts the back room they spawn in with extra loot and slight fortifications."```

#### Purpose of change
Start populating the city with new NPCs.  This NPC spawns with a slightly fortified variation of the back room of s_restaurant.json.   Depending on feedback, I can make some more.

#### Describe the solution
Adds an NPC chef that has remained at their workplace since the cataclysm, living off of the supplies.  

#### Describe alternatives you've considered
Not touching npcs.

#### Additional context
Chef NPC, spawns with chefs clothing and a butchers knife.  Chance of higher cooking and throwing skills.  They are pretty likely to go pick a fight if they hear zombies, so that's interesting.  They also spawn with a generous hoard of food, mostly perishable.  Once ownership becomes functional, I can adjust the NPC to own the loot.

Backroom with NPC spawn & without.
NPC chef spawn
npc free

### CODE REVIEW

**Comment by ifreund** [comment]
This is pretty sweet, I think it would be awesome to see the world populated with more, flavorful static NPCs like this one

**Comment by Night-Pryanik** [comment]
Yeah, that's something I wanted to do myself sometime.

File: data/json/npcs/TALK_SURVIVOR_CHEF.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF",
+    "dynamic_line": "Hey, I didn't expect to live long enough to see another living human!",
+    "responses": [
+      { "text": "What are you doing here?", "topic": "TALK_NC_SURVIVOR_CHEF_INTRO" },
+      { "text": "Wanna get outta here?.", "topic": "TALK_SUGGEST_FOLLOW" },
```
Comment (Night-Pryanik) [inline data/json/npcs/TALK_SURVIVOR_CHEF.json]:
```suggestion
      { "text": "Wanna get outta here?", "topic": "TALK_SUGGEST_FOLLOW" },
```

File: data/json/npcs/TALK_SURVIVOR_CHEF.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_LEAVE",
+    "dynamic_line": "Well, the dishwasher made a break for it three days after things got weird.  He was ripped to shreads before he made it to the street.  I figure this place has gotta be safer then my apartment, and at least I've got all this food here.",
+    "responses": [
+      { "text": "What did you do before the cataclysm?", "topic": "TALK_NC_SURVIVOR_CHEF_EXPERTISE" },
+      { "text": "Wanna get outta here?.", "topic": "TALK_SUGGEST_FOLLOW" },
```
Comment (Night-Pryanik) [inline data/json/npcs/TALK_SURVIVOR_CHEF.json]:
```suggestion
      { "text": "Wanna get outta here?", "topic": "TALK_SUGGEST_FOLLOW" },
```

File: data/json/npcs/TALK_SURVIVOR_CHEF.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_EXPERTISE",
+    "dynamic_line": "I've been a cook since forever, this wasn't the best joint, but management was cool.",
+    "responses": [
+      { "text": "Why don't you go somewhere else?", "topic": "TALK_NC_SURVIVOR_CHEF_LEAVE" },
+      { "text": "Wanna get outta here?.", "topic": "TALK_SUGGEST_FOLLOW" },
```
Comment (Night-Pryanik) [inline data/json/npcs/TALK_SURVIVOR_CHEF.json]:
```suggestion
      { "text": "Wanna get outta here?", "topic": "TALK_SUGGEST_FOLLOW" },
```

**Comment by curstwist** [comment]
> 
> 
> Yeah, that's something I wanted to do myself sometime.

I-am-Erk has the same idea too, so I guess it's time has come :).  This should give a rough template for others to follow.  Someone can also improve its dialogue later, which isn't my strength.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/faction-a-benign-religious-sect/19668/2

**Comment by Mecares** [comment]
Maybe add a hostile cannibal variant, that has a rather gruesome hoard of preserved food.

**Pull request merged**